### PR TITLE
Adapt input format

### PR DESCRIPTION
**Pull request by mackaiver**
This changes the input format a bit. Now contains more information. Especially MC related things. 
Also changes some names of keys in the data item. I also added a class for mapping key names to constants. Don't know if thats a good idea.

### CODE REVIEW

File: src/main/java/streams/cta/io/ImageStream.java [hunk 1]
```diff
@@ -26,13 +22,26 @@
     public ImageStream(SourceURL url) {
         super(url);
     }
-
     public ImageStream() {
     }
 
+
+    private class MC {
```
Comment (alexeyegorov) [inline src/main/java/streams/cta/io/ImageStream.java]: some minor comment about these private classes would be nice (although, most of the things are clear at the moment, but for the purity of the project!)

File: src/main/java/streams/cta/io/ImageStream.java [hunk 2]
```diff
@@ -26,13 +22,26 @@
     public ImageStream(SourceURL url) {
         super(url);
     }
-
     public ImageStream() {
     }
 
+
```
Comment (alexeyegorov) [inline src/main/java/streams/cta/io/ImageStream.java]: some minor comment about these private classes would be nice (although, most of the things are clear at the moment, but for the purity of the project!)

File: src/main/java/streams/cta/io/ImageStream.java [hunk 3]
```diff
@@ -53,15 +62,23 @@ public Data readNext() throws Exception {
             return null;
         }
```
Comment (alexeyegorov) [inline src/main/java/streams/cta/io/ImageStream.java]: this looks more readable for me! 👍

File: src/main/java/streams/cta/io/ImageStream.java [hunk 4]
```diff
@@ -26,13 +22,26 @@
...
+        int[] triggeredTelescopes;
+        int numTriggeredTelescopes;
+    }
+    private class Event{
+        Map<Integer, double[]> images;
+        MC mc;
+        Array array;
+        long eventId;
+    }
+
```
Comment (alexeyegorov) [inline src/main/java/streams/cta/io/ImageStream.java]: some minor comment about these private classes would be nice (although, most of the things are clear at the moment, but for the purity of the project!)

File: src/main/java/streams/cta/io/ImageStream.java (hunk 3 above)
Comment (alexeyegorov) [inline src/main/java/streams/cta/io/ImageStream.java]: sorry, this comment function is not as I would have awaited it to be (the "good looking" code is below the marked position!)

File: src/main/java/streams/cta/io/ImageStream.java (hunk 4 above)
Comment (mackaiver) [inline src/main/java/streams/cta/io/ImageStream.java]: Added a bunch of comments

**Pull request merged**